pages/api/openPR.js

```diff
@@ -249,11 +250,11 @@ export default async (req, res) => {
         parseProjectName(values),
         getSubmissionFiles(values, nomineeJSON)
       )
-    ).then((results) => {
+    ).then(async (results) => {
       let pullRequestNumbers = [];
 
       // Only returning successful results in an array of PR number(s)
-      results.forEach(async (result) => {
```

Comment:
`forEach` spawns an async process for each item in the array and the loop cannot be `break`. Thus, the loop has been replaced with `for ... of...` that then can take a `break` when an error is found.